components/core/tools/docker-images/clp-env-base-manylinux-2.28-aarch64/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM quay.io/pypa/manylinux_2_28_aarch64
```

Hadolint (2.12.0)

[warning] Always tag the version of an image explicitly

(DL3006)